File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    const answer = [];
    for ([i,j,k] of commands) {
        const arrS = array.slice(i-1,j);
        arrS.sort((a,b)=>a-b);
        // console.log(arrS);
        answer.push(arrS[k-1])
    };
    return answer
}

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for (int i=0; i<commands.length; i++) {
            int s=commands[i][0], e=commands[i][1], k=commands[i][2];
            int[] temp = Arrays.copyOfRange(array, s-1, e);
            Arrays.sort(temp);
            answer[i] = temp[k-1];
        }
        
        return answer;
    }
}